client/src/app/auth/vuex/actions.js

```diff
@@ -0,0 +1,59 @@
+// plugins and utils are alias. see client/build/webpack.base.conf.js
+import { http, setToken as httpSetToken } from 'plugins/http'
+import { isEmpty } from 'lodash'
+import { getData } from 'utils/get'
+import { localStorageSetItem } from 'utils/local'
+import * as TYPES from './mutations-types'
+
+export const attemptLogin = ({ dispatch }, { email, password }) => http.post('/login', { email, password })
+     // functional approach, more readable and generate minus code
+     // .then(response => response.data)
+     // .then({ data } => data)
+     // .then(response => getData(response))
+    .then(getData)
```

Comment:
Not to me. I had to debug it to understand which param it receives and what it extract and returns. We have to keep in mind this project is also a learning resource. Things that are not obvious should be explained!